services/motion/builtin/move_request.go

```diff
@@ -95,55 +93,19 @@ func (mr *moveRequest) Plan(ctx context.Context) (state.PlanResponse, error) {
 	// TODO(RSDK-5634): this should pass in mr.seedplan and the appropriate replanCostFactor once this bug is found and fixed.
 	plan, err := motionplan.Replan(ctx, mr.planRequest, nil, 0)
 	if err != nil {
-		return state.PlanResponse{}, err
-	}
-
-	waypoints, err := plan.GetFrameSteps(mr.kinematicBase.Kinematics().Name())
-	if err != nil {
-		return state.PlanResponse{}, err
-	}
-
-	switch mr.requestType {
-	case requestTypeMoveOnMap:
```

Comment:
Why do we not need to care about whether we are doing moveOnMap or moveOnGlobe anymore?

Reply:
Everything that was being done here was to rectify the plan.  In the case of MoveOnGlobe we are also smuggling geoposes.  I want to only smuggle them out at the point they are being returned rather than internally storing them as this representation.  So the need for cased code falls out, since the Plan type does it all.